src/filesystem/interface.js

```diff
@@ -340,8 +340,7 @@ function FileSystem(options, callback) {
     { name: 'ftruncate' },
     { name: 'futimes' },
     { name: 'getxattr', promises: true, absPathArgs: [0] },
-    // lchown - https://github.com/filerjs/filer/issues/620
-    // lchmod - https://github.com/filerjs/filer/issues/619
+    { name: 'lchown' },
```

Comment:
This should be `{ name: 'lchown', promises: true },` so that it automatically adds a `.promises.lchown` version for us.